src/redux/boards/actions.js

```diff
@@ -64,6 +67,33 @@ const actionCreators = {
       type: actions.GET_BOARD_FAILURE,
       payload: { error }
     };
+  },
+  deleteBoard(boardId) {
+    return async dispatch => {
+      dispatch({ type: actions.DELETE_BOARD });
+      try {
+        const response = await BoardService.deleteBoard(boardId);
+        if (response.ok) {
+          dispatch(actionCreators.deleteBoardSuccess(boardId));
+        } else {
+          throw new Error(response);
+        }
+      } catch (e) {
```

Comment:
Here we are catching that `throw new Error`. I'd remove this `catch` and dispatch `deleteBoardFailure` in the `else` clause directly.

Reply:
we are also rejections in the `deleteBoard` promise. We need this